bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java

```diff
@@ -190,7 +190,7 @@ private void sendResponse(InputStream stream, long size, String mimeType, String
             response.setContentLength((int) size);
         }
         if (inline) {
-            response.setHeader(CONTENT_DISPOSITION_HEADER, "inline");
+            response.setHeader(CONTENT_DISPOSITION_HEADER, INLINE_SELECTOR);
```

Comment:
I would not use `INLINE_SELECTOR` as it represents a request selector, which has a different meaning than the content disposition header which is used here. You might want to introduce a new constant called e.g. `CONTENT_DISPOSTION_INLINE`.